test/tests/mortar/mortar-q-points/test.i

```diff
@@ -0,0 +1,69 @@
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    file = nodal_normals_test_offset_nonmatching_gap.e
+  []
+  [./primary]
+    input = file
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '2'
+    new_block_id = '20'
+  [../]
+  [./secondary]
+    input = primary
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '1'
+    new_block_id = '10'
+  [../]
+  uniform_refine = 2
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
```

Comment:
Why you turned this off? Because block 20 and 10 do not have kernels?

Reply:
Hmm I may be able to remove this

Reply:
I actually cannot. In most mortar applications a user will have Lagrange Multiplier variable that lives on the secondary block, so I do have a coverage check that makes sure that there is variable defined on the secondary block (block '10' in this case; we do not check to see that there is a variable defined on the primary block). So if I remove this, then I get this error:
```
*** ERROR ***
Each subdomain must contain at least one Kernel.
The following block(s) lack an active kernel: 10 
```
However, it is a valid use case for mortar to not have an LM variable, but I would like users to have to explicitly say something in order for MOOSE not to error, in order to indicate that they know what they're doing.

Reply:
I feel like the same thing can be implemented by the interface kernel as well. Using mortar is of course still good

Reply:
The domains are disconnected here (and even if they were the mesh would be non-conforming) so interface kernel would not work. Only beautiful mortar works here